src/win32.mak

```diff
@@ -578,8 +578,8 @@ $G/gsroa.obj : $C\gsroa.d
 $G/md5.obj : $C\md5.d
 	$(HOST_DC) -c -of$@ $(DFLAGS) -betterC -mv=dmd.backend=$C $C\md5
 
-$G/mscoffobj.obj : $C\mscoff.h $C\mscoffobj.c
-	$(CC) -c -o$@ $(MFLAGS) -I$D;$(ROOT) -I$G $C\mscoffobj
+$G/mscoffobj.obj : $C\mscoff.d $C\mscoffobj.d
```

Comment:
This compiles the 2 files into different object files, but not both are passed to the linker. Simplest to compile them seperately.

Reply:
One wonders why that suddenly stopped working (and it works on my machine). Sigh.